python/ray/autoscaler/aws/example-full.yaml

```diff
@@ -25,6 +25,10 @@ autoscaling_mode: default
 docker:
     image: "" # e.g., tensorflow/tensorflow:1.5.0-py3
     container_name: "" # e.g. ray_docker
+    # If true, pulls latest version of image. Otherwise, `docker run` will only pull image
+    # if no cached version is present. You may want to disable this if you are keeping the
```

Comment:
remove second sentence, advanced users will know to disable it and it doesn't cause much harm if they don't